lib/network.py

```diff
@@ -1076,13 +1077,45 @@ def synchronous_get(self, request, timeout=30):
     def broadcast(self, tx, timeout=30):
         tx_hash = tx.txid()
         try:
-            out = self.synchronous_get(('blockchain.transaction.broadcast', [str(tx)]), timeout)
+            out = self.synchronous_send(('blockchain.transaction.broadcast', [str(tx)]), timeout)
         except BaseException as e:
             return False, "error: " + str(e)
         if out != tx_hash:
             return False, "error: " + out
         return True, out
 
+    def get_history_for_scripthash(self, hash):
+        command = 'blockchain.scripthash.get_history'
+        return (command, [hash])
+
+    def subscribe_to_headers(self):
+        command = 'blockchain.headers.subscribe'
+        return (command, [])
+
+    def subscribe_to_address(self, address):
+        command = 'blockchain.address.subscribe'
+        return (command, [address])
+
+    def get_merkle_for_transaction(transaction_hash, transaction_height):
```

Comment:
what about `self`? or maybe you want this to be a `staticmethod`?

Reply:
You are 100% right. I realized that when my work progressed and I've addressed that in #4376, https://github.com/spesmilo/electrum/pull/4376/files#diff-08f47fc4236c4931079fa1058b996bb2R1128